src/catkin_pkg/templates/metapackage.cmake.in

```diff
@@ -1,4 +1,4 @@
 cmake_minimum_required(VERSION 2.8.3)
-project(@name@)
+project(@name)
```

Comment:
Ok, I see you already assume the template is given with the path [here](https://github.com/ros-infrastructure/catkin_pkg/blob/752c549237c1aaf04731967af4f46db20de71e9c/src/catkin_pkg/cmake.py#L54). 

But I was wondering, ain't better to add something similar to [this](https://github.com/ros-infrastructure/catkin_pkg/blob/752c549237c1aaf04731967af4f46db20de71e9c/src/catkin_pkg/package_templates.py#L145) in that function? Do you think it might cause a problem?